Plots/Bar/NCL_bar_14.py

```diff
@@ -0,0 +1,83 @@
+"""
+NCL_bar_14.py
+===========
+This script illustrates the following concepts:
+    - Drawing an XY curve on top of filled bars
+    - Filling the bars in a bar plot with different colors
+    - Setting the minimum/maximum value of the X and Y axis in a bar plot
+    - Rotating the Y axis string 270 degrees
+    - Drawing a Y reference line in an XY plot
+    - Changing the color and thickness of a Y reference line
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/bar_14.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/bar_14_lg.png
+"""
+###############################################################################
+# Import packages
+
+import numpy as np
+import matplotlib.pyplot as plt
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Generate data:
+
+# Generate 12 numbers from 0 to 100 in a uniform distribution
+y1 = np.random.uniform(0, 100, 12)
+
+# Generate 12 numbers with an average of 30 and a standard deviation of 10
+y2 = np.random.normal(30, 10, 12)
+
+# Create an array from 1 to 12 with 12 numbers
+months = np.linspace(1, 12, 12)
+
+###############################################################################
+# Plot:
+
+# Create figure with figure size (width, height) in inches and create axes
+fig = plt.figure(figsize=(6, 5.25))
+ax = plt.axes()
+
+# Plot the bar chart, line chart, and a horizontal line with varying colors and linewidths
+plt.bar(months, y1, color="yellow", edgecolor="k", linewidth=.5)
+plt.plot(months, y2, color="b", linewidth=1)
+plt.plot([0, 13], [40, 40], color="b", linewidth=1)
```

Comment:
There's actually a way to plot horizontal lines in matplotlib. See the [hlines documentation](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.hlines.html).

Reply:
Just fixed it to use hlines down below! Thanks for the help!